client/src/components/_common/Button/Button.module.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
**🛎  Sad Empty File**
This file should have or load the styles for the new button component.

<details><summary><strong>Options</strong></summary>

<details><summary><strong>A.</strong> Add classes here, <em>copying</em> what you need from <code>c-button.css</code>.</summary>

```
.container {
  ... /* copy from `.c-button` */
}

.primary {
  ... /* copy from `.c-button--primary` */
}
.secondary {
  ... /* copy from `.c-button--secondary` */
}

.as-link {
  ... /* copy from `.c-button--as-link` */
}

...
```

</details>
<details><summary><strong>B.</strong> Add classes here, "composing" what you need from <code>c-button.css</code>.</summary>

```
.container {
  composes: c-button from '../../../styles/components/c-button.css';
}

.primary {
  composes: c-button--primary from '../../../styles/components/c-button.css';
}
.secondary {
  composes: c-button--secondary from '../../../styles/components/c-button.css';
}

.as-link {
  composes: c-button--as-link from '../../../styles/components/c-button.css';
}

...
```

- Examples: (simple) [`Pill.module.scss`](https://github.com/TACC/Core-Portal/blob/2c0fe78/client/src/components/_common/Pill/Pill.module.scss), (complex) [`Message.module.scss`](https://github.com/TACC/Core-Portal/blob/2c0fe78/client/src/components/_common/Message/Message.module.scss).
- Note: Use `.container` (not `.root`) for the containing/base/root element.

</details>

<details><summary>(I am sorry for any earlier confusion.)</summary>

I may have mislead you before in [a Slack comment](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1641930563005500?thread_ts=1641928526.004400&cid=CQUA4D5KJ).

_I am comparatively new to CSS Modules. The classes I proposed were suitable for global CSS (which should have unique, re-usable classnames), but unnecessarily verbose for CSS modules (which can be simple, one-off class names). My brain thinks in global CSS first._

</details>

</details>
<details><summary>ES Lint Error</summary>

```
➜  client git:(task/FP-545) npm run lint

> client@2.0.0 lint
> npm run lint:js && npm run lint:css && npm run prettier:check


> client@2.0.0 lint:js
> eslint . --ext .js,.jsx,.ts,.tsx


> client@2.0.0 lint:css
> stylelint '**/*.{css,scss,sass}'


src/components/_common/Button/Button.module.scss
 1:1  ✖  Unexpected empty source   no-empty-source
```

</details>